code/game/objects/items/weapons/tools.dm

```diff
@@ -193,7 +193,7 @@
 		item_state = "cutters_yellow"
 
 /obj/item/weapon/wirecutters/attack(mob/living/carbon/C as mob, mob/user as mob)
-	if((iscarbon(C)) && (C.handcuffed) && (istype(C.handcuffed, /obj/item/weapon/handcuffs/cable)))
+	if((iscarbon(C)) && (C.handcuffed) && (istype(C.handcuffed, /obj/item/weapon/handcuffs/cable)) && user.a_intent!=I_HARM)
```

Comment:
needs to be cast as a human to get intent

Reply:
i assumed that intent was lower down than human
how would this work for borgs then?

Reply:
[DNM]

Reply:
My bad, intent is at a mob level

You messed up the define, it's I_HURT not I_HARM

Reply:
oh no i figured it out it's HURT not HARM

Reply:
wew i didn't refresh the page yes we both got it

Reply:
heck yeah